operations/node_management/Add_Remove_Replace_NCNs/Redeploy_Services.md

```diff
@@ -0,0 +1,270 @@
+    ncn-mw# echo $SYSMGMT_VERSION
+    ```
+
+2. Determine the current resources and retention settings:
+
+    ```bash
+    ncn-mw# kubectl -n loftsman get cm loftsman-platform -o jsonpath='{.data.manifest\.yaml}' | yq r - 'spec.charts.(name==cray-sysmgmt-health).values.prometheus-operator.prometheus.prometheusSpec.resources'
+    ```
+
+    Example Output:
+
+    ```
+    limits:
+      cpu: '6'
+      memory: 30Gi
+    requests:
+      cpu: '2'
+      memory: 15Gi
+    ```
+
+    ```bash
+    ncn-mw# kubectl -n loftsman get cm loftsman-platform -o jsonpath='{.data.manifest\.yaml}' | yq r - 'spec.charts.(name==cray-sysmgmt-health).values.prometheus-operator.prometheus.prometheusSpec.retention'
+    ```
+
+    Example Output:
+
+    ```
+    48h
+    ```
+
+3. Create `sysmgmt-health-manifest.yaml` and update the `resources` and `retention` sections as needed based up the data from the above step:
+
+    ```bash
+    ncn-mw# cat > sysmgmt-health-manifest.yaml << EOF
+    apiVersion: manifests/v1beta1
+    metadata:
+        name: sysmgmt-health
+    spec:
+        charts:
+        - name: cray-sysmgmt-health
+          version: $SYSMGMT_VERSION
+          namespace: sysmgmt-health
+          values:
+            prometheus-operator:
+              prometheus:
+                prometheusSpec:
+                  resources:
+                    limits:
+                      cpu: '6'
+                      memory: 30Gi
+                    requests:
+                      cpu: '2'
+                      memory: 15Gi
+                  retention: 48h
+    EOF
+    ```
+
+3. Merge `customizations.yaml` with `sysmgmt-health-manifest.yaml`:
+
+    ```bash
+    ncn-mw# manifestgen -c /tmp/customizations.yaml -i sysmgmt-health-manifest.yaml > sysmgmt-health-manifest.out.yaml
+    ```
+
+4. Redeploy the SYSMGMT_HEALTH helm chart:
+
+    ```bash
+    ncn-mw# loftsman ship \
+        --charts-repo https://packages.local/repository/charts \
+        --manifest-path sysmgmt-health-manifest.out.yaml
+    ```
+
+5. Check that the new endpoint has been updated.
+
+    ```bash
+    ncn-mw# kubectl get endpoints -l app=cray-sysmgmt-health-ceph-exporter -n sysmgmt-health -o jsonpath='{.items[*].subsets[].addresses}' | jq -r '.[] | .ip'
+    ncn-mw# kubectl get endpoints -l app=cray-sysmgmt-health-ceph-node-exporter -n sysmgmt-health -o jsonpath='{.items[*].subsets[].addresses}' | jq -r '.[] | .ip'
+    ```
+
+    Example output:
+
+    ```
+    10.252.1.13
+    10.252.1.4
+    10.252.1.5
+    10.252.1.6
+    ``` 
+
+#### 1.5 Cleanup
+ 
+1. Remove temporary files
+
+   ```bash
+   ncn-mw# rm /tmp/customizations.yaml /tmp/customizations.original.yaml /tmp/customizations.original.yaml.pretty
+   ```
+
+Proceed to the next step:
+
+- Added Storage NCN : [Validate NCN](Validate_NCN.md) or return to the main [Add, Remove, Replace or Move NCNs](Add_Remove_Replace_NCNs.md) page.
```

Comment:
both of these 'return to main' links are 404

Reply:
fixed